lang-perl/perl-data-perl/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=perl-data-perl
+PKGSEC=perl
+PKGDES="Base classes wrapping fundamental Perl data types"
```

Comment:
Fundamental Perl data types for wrapping base classes

... or ...

Base class wrappers for fundamental Perl data types